examples/svelte/svelte-vite/yarn.lock

```diff
@@ -0,0 +1,1924 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Can you delete this yarn file? We want our examples to be as simple as possible, so we only use `npm install` in them